modules/rosa-cluster-autoscaler-cli-edit.adoc

```diff
@@ -8,7 +8,30 @@
 
 You can edit any specific parameters of the cluster autoscaler after creating the autoscaler.
 
-- To edit the cluster autoscaler, run the following command:
+.Procedure
+
+//ROSA HCP procedure
+ifdef::openshift-rosa-hcp[]
+. To edit the cluster autoscaler, run the following command:
```

Comment:
Since this is a single step, should it be an unordered list instead of ordered?

Also, is the substep supposed to be a substep? This step seems to be something that you could do independent of the first step, not something that's dependent on the first step.

Reply:
Yeah, you're right. Updating